From lrodriguez@Atheros.com  Thu Jan 22 14:31:26 2009
From: Luis R. Rodriguez <lrodriguez@atheros.com>
Date: Tue, 20 Jan 2009 11:42:43 -0800
Subject: ath9k: quiet harmless ForceXPAon messages
To: Paul Hickey <paul@christianpatriot.us>, <stable@kernel.org>, Kyle McMartin <kyle@mcmartin.ca>
Cc: linux-wireless@vger.kernel.org, Luis Rodriguez <Luis.Rodriguez@Atheros.com>
Message-ID: <20090120194243.GC19581@tesla>
Content-Disposition: inline


From: Luis R. Rodriguez <lrodriguez@atheros.com>

This is a port of one line of upstream patch
f1dc56003b23d2d5bb5a756de6b1633a76c9e697

The "ForceXPAon" messages on ath9k were not meant to be printed
regularly, lets quiet them as this can happen quite frequently
(scans) and will fill the logs with tons of these messages.

Signed-off-by: Sujith <Sujith.Manoharan@atheros.com>
Signed-off-by: Luis R. Rodriguez <lrodriguez@atheros.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/net/wireless/ath9k/hw.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/net/wireless/ath9k/hw.c
+++ b/drivers/net/wireless/ath9k/hw.c
@@ -729,7 +729,7 @@ ath9k_hw_eeprom_set_board_values(struct 
 					  AR_AN_TOP2_LOCALBIAS,
 					  AR_AN_TOP2_LOCALBIAS_S,
 					  pModal->local_bias);
-		DPRINTF(ah->ah_sc, ATH_DBG_ANY, "ForceXPAon: %d\n",
+		DPRINTF(ah->ah_sc, ATH_DBG_EEPROM, "ForceXPAon: %d\n",
 			pModal->force_xpaon);
 		REG_RMW_FIELD(ah, AR_PHY_XPA_CFG, AR_PHY_FORCE_XPA_CFG,
 			      pModal->force_xpaon);
